apps/dashboard/src/app/account/billing/checkout/checkout.component.html

```diff
@@ -1,7 +1,33 @@
 <div class="checkout">
   <div>
     <h2>{{ 'billing.payment_details' | translate }}</h2>
-    <div class="body-m">{{ 'billing.payment_details.description' | translate }}</div>
+  </div>
+
+  <div>
+    <div class="section-header">
+      <div class="section-title">
+        <pa-icon
+          name="circle-check"
+          color="#f06"></pa-icon>
```

Comment:
FYI you can also change icon color simply by setting the color attribute in CSS like 
```scss
.section-title pa-icon {
  color: $color-primary-regular;
}
```

Reply:
Basically, `pa-icon` inherit its color from its parent. You could also do
```scss
.section-title {
  color: $color-primary-regular;
}
```
if everything in `.section-title` shared the same color.